starboard/win/win32/test_filters.py

```diff
@@ -59,6 +59,9 @@
 
         # Enable once verified on the platform.
         'SbMediaCanPlayMimeAndKeySystem.MinimumSupport',
+
+        # TODO: b/349109647 Disable flaky test.
+        'SbPlayerGetMediaTimeTests',
```

Comment:
I think this should be a wildcard here, no ? `SbPlayerGetMediaTimeTests.*` ?

Reply:
Fixed.